src/main/java/org/sonar/plugins/stash/issue/CoverageIssuesReport.java

```diff
@@ -0,0 +1,78 @@
+package org.sonar.plugins.stash.issue;
+
+import java.util.Collection;
+
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.collections.Predicate;
+import org.apache.commons.lang3.StringUtils;
+import org.sonar.plugins.stash.StashPluginUtils;
+
+
+public class CoverageIssuesReport extends Report {
+  
+  private double previousProjectCoverage;
+  
+  public CoverageIssuesReport() {
+    super();
+    this.previousProjectCoverage = 0;
+  }
+  
+  public double getPreviousProjectCoverage() {
+    return previousProjectCoverage;
+  }
+
+  public void setPreviousProjectCoverage(double previousProjectCoverage) {
+    this.previousProjectCoverage = previousProjectCoverage;
+  }
+  
+  public boolean isEmpty() {
+    return (Double.valueOf(getProjectCoverage()).intValue() == 0) && getIssues().isEmpty();

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the boxing to "Double". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2153)